labonneboite/common/mapping.py

```diff
@@ -105,76 +105,75 @@ def get_affinity_between_rome_and_naf(rome_code, naf_code):
     return 1.0 * current_rome_hirings / total_naf_hirings
 
 
-class Rome2NafMapper(object):
-
-    def __init__(self):
-        self.rome_2_naf_dict = MANUAL_ROME_NAF_MAPPING
-
-    def map(self, rome_codes, optional_naf_codes=None):
-        naf_codes = set()
-        for rome in rome_codes:
-            if rome not in settings.ROME_DESCRIPTIONS.keys():
-                raise Exception('bad rome code %s' % rome)
-            try:
-                naf_codes_with_hirings = self.rome_2_naf_dict[rome]
-            except KeyError:
-                logger.error('soft fail: no NAF codes for ROME %s', rome)
-                naf_codes_with_hirings = {}
-            for naf, _ in naf_codes_with_hirings.iteritems():
-                if optional_naf_codes:
-                    if naf in optional_naf_codes:
-                        naf_codes.add(naf)
-                else:
+def map_romes_to_nafs(rome_codes, optional_naf_codes=None):
+    naf_codes = set()
+    for rome in rome_codes:
+        if rome not in settings.ROME_DESCRIPTIONS.keys():
```

Comment:
Ici on fait une recherche dans une liste. Préférer : `if rome not in settings.ROME_DESCRIPTIONS`
(même si ce n'est pas du nouveau code)

Reply:
will do